src/lib/components/SaveButton.js

```diff
@@ -33,7 +31,7 @@ export class SaveButtonComponent extends Component {
       >
         {(formik) => (
           <>
-            { ( formik.isSubmitting && formState === FORM_SAVING ) ? (
+            {formik.isSubmitting && formState === FORM_SAVING ? (
```

Comment:
This is Prettier, and I don't want to think even for a second about code formatting.
I want to write my code in an ugly way, hit CTRL+S and let something else formatting it, in the same way as it would be formatted for others devs. As for Python when using Black.
I strongly suggest everyone to configure their IDE to do this and integrate ES linting too, there are many unused imports and other small mistakes around that are reported by the IDE.
I will bring this topic up in the next teams meeting.